### ISSUE TITLE
[MHV Portal / Accessibility] Breadcrumb nav scrolling inconsistent / unneeded

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#130509]
### 📝 User story

As a user, I want to select the current page link in the breadcrumbs and have the normal action of the H1 focus only, without page scroll to the MHV sub-navigation.

### 📝 Problem statement

Selecting the current page link in the breadcrumbs "reloads" the current page, sets focus to the H1, and scrolls the page to the MHV secondary nav.  

Do not change a user's context when an item gains focus. In this case, we imply a context change by limiting the view of the main navigation and moving the menu to the MHV navigation. This action is potentially confusing to the user and should be removed.

### 🌌 Supporting screenshots, videos, code snippets

https://github.com/user-attachments/assets/30f49268-525d-48d1-a947-678da91dc955

### 👣 Steps to reproduce the issue

1. Navigate to MHV
2. Navigate to any / all landing pages 
3. Click on the last link in the breadcrumb navigation (current page)
4. >>  Notice defect action: scrolling to MHV secondary navigation

### 🔗 Point of contact

@steveg-IAT

### ⚠️ Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### 📝 Related issues

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/123109

### 🔎 Accessibity standards

- On Focus (Level A) : [SC 3.2.1](https://w3.org/TR/WCAG22#on-focus): Do not change a user's context when an item gains focus

### 𝌣 Proposed solution or next steps

- [ ] Remove the scrolling functionality across all pages
- [ ] Add code snippet of offending code

### ✅ Acceptance criteria

- [ ] Clicking on the last breadcrumb (current page) element does not scroll the page, but does set focus to the page H1.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to "QA/A11y Review" after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.
